### PR TITLE
SDL_migration: add RW read/write

### PR DESCRIPTION
**Pull request by 1bsyl**

### CODE REVIEW

File: build-scripts/SDL_migration.cocci [hunk 1]
```diff
@@ -27,10 +27,52 @@
...
 
+// SDL_RWread
+@@
+expression e1, e2, e3, e4;
+identifier i;
+@@
+(
+   i = SDL_RWread(e1, e2,
+-  e3, e4);
++  e3 * e4);
```
Comment (madebr) [inline build-scripts/SDL_migration.cocci:42]: Will `SDL_RWread(rwops, data, 4 + 4, 5 + 5)` be converted correctly?

Comment (1bsyl) [reply build-scripts/SDL_migration.cocci:42]: yes, I think so :)

Comment (1bsyl) [reply build-scripts/SDL_migration.cocci:42]: this isn't a "preprocessor", it really analyse the code .

Comment (madebr) [reply build-scripts/SDL_migration.cocci:42]: Yup, you're right!
```patch
-SDL_RWread(rwops, data, 4 + 4, 5 + 5);
+SDL_RWread(rwops, data, (4 + 4) * (5 + 5));
```

Comment (slouken) [reply build-scripts/SDL_migration.cocci:42]: You might want to special case where one of the two is `1`, as that can just be removed.

Comment (1bsyl) [reply build-scripts/SDL_migration.cocci:42]: yes simplification here: 
https://github.com/libsdl-org/SDL/commit/9eaea7d6614c00c865c8059a161631a2668adf12

**Pull request merged**